Changes in module 'nf-core/modules/untar'
--- modules/nf-core/modules/untar/main.nf
+++ modules/nf-core/modules/untar/main.nf
@@ -24,6 +24,8 @@
 
     """
     mkdir output
+
+    echo "hello world"
 
     ## Ensures --strip-components only applied when top level of tar contents is a directory
     ## If just files or multiple directories, place all in output

************************************************************
